tests/device_tests/ethereum/test_definitions_bad.py

```diff
@@ -7,7 +7,7 @@
 from trezorlib import ethereum
 from trezorlib.debuglink import TrezorClientDebugLink as Client
 from trezorlib.exceptions import TrezorFailure
-from trezorlib.messages import EthereumDefinitionType
+from trezorlib.messages import DefinitionType
```

Comment:
I'm not sure if some or perhaps all of the tests shouldn't be moved under `device_tests/common/test_definitions_bad.py` as they're not really ethereum specific anymore. WDYT?

(maybe also check `tests/device_tests/ethereum/test_definitions.py` if those shouldn't be moved under `common` as well and extended with Solana tests.

Reply:
It would make sense to move them to common, probably all _bad tests. We can also move out all the test_definitions.py tests as well just so they're in the same place